mods/mapgen/oregen.lua

```diff
@@ -576,17 +597,6 @@ function mapgen.register_ores()
 		y_min          = 1025,
 	})
 
-	minetest.register_ore({
-		ore_type       = "scatter",
-		ore            = "blocks:stone_with_coal",
-		wherein        = "blocks:stone",
-		clust_scarcity = 8 * 8 * 8,
-		clust_num_ores = 8,
-		clust_size     = 3,
-		y_max          = 64,
-		y_min          = -127,
-	})
-
```

Comment:
any reason why this was removed?

Reply:
Accidental, this PR shouldn't edit any default code.